### ISSUE TITLE
How to setup a web config to access  from outside?

### ISSUE THREAD

**Issue by jeong760** [bonesoul/CoiniumServ#1007]
@bonesoul 

I've configured a website of CoiniumServ as following but could not possible to access. so  is there any sample for  this

- Created a DNS A record on Godaddy.com, which point to my server public IP

- Open and set a policy on firewall to allow access a website form external 

- Configured a website like a following

**Website configuration on config.json**
 "website": {
        "enabled": true,
        "bind": "192.168.112.100",  <--- this is a servers private IP address. 
        "port": 80,
        "template": "default",
        "feed": "",
        "backend": {
            "metrics": {
                "enabled": false
            }
        }

**Comment by jeong760** [comment]
@bonesoul Also can I set a SSH for CoiniumServ website?

**Comment by bonesoul** [comment]
you mean https?

**Comment by jeong760** [comment]
@bonesoul, yes, I want to implement a SSH(https) conneciton on default website.

**Comment by bonesoul** [comment]
ssh is different then https. you should check https://github.com/NancyFx/Nancy/wiki/

**Comment by jeong760** [comment]
@bonesoul  Thank  you.  Have a  nice weekend..
